src/sentry/middleware/integrations/parsers/jira.py

```diff
@@ -41,25 +38,24 @@ class JiraRequestParser(BaseRequestParser):
     outbox_response_region_classes = [JiraIssueUpdatedWebhook]
 
     def get_integration_from_request(self) -> Integration | None:
-        try:
-            return parse_integration_from_request(request=self.request, provider=self.provider)
-        except AtlassianConnectValidationError as e:
```

Comment:
What catches this error now?

Reply:
I was thinking that raising it might be appropriate, but I just noticed that some endpoints don't get validated like this, so I'm gonna put back the error catching.